src/test/java/api/UserApi.java

```diff
@@ -0,0 +1,30 @@
+package api;
+
+import com.github.javafaker.Faker;
+import io.restassured.response.Response;
+
+public class UserApi extends ApiBase {
+    RegistrationDTO dto;
+    Response response;
+    Faker faker = new Faker();
+
+    public RegistrationDTO randomUserData() {
+        dto = new RegistrationDTO();
+        dto.setFull_Name(faker.name().firstName());
+        dto.setEmail(faker.internet().emailAddress());
```

Comment:
the better if you random data faker.internet().emailAddress() transfer to variable 
String email = faker.internet().emailAddress()
under class , where variable faker